docs/pages/reference/backends.mdx

```diff
@@ -409,10 +406,47 @@ High Availability. DynamoDB backend supports two types of Teleport data:
 - Audit log events
 
 DynamoDB cannot store the recorded sessions. You are advised to use AWS S3 for
-that as shown above. To configure Teleport to use DynamoDB:
+that as shown above. 
+
+### Authenticating to AWS
+
+The Teleport Auth Service must be able to read AWS credentials in order to
+authenticate to DynamoDB. 
+
+(!docs/pages/includes/aws-credentials.mdx service="the Teleport Auth Service"!)
+
+The IAM role that the Teleport Auth Service authenticates as must have the
+policies specified in the next section.
+
+### IAM policies
+
+Make sure that the IAM role assigned to the Teleport Auth Service is configured
+with sufficient access to DynamoDB. Below is the example of the IAM policy you
+can use: 
+
+```js
+{
+    "Version": "2012-10-17",
+    "Statement": [{
+            "Sid": "AllAPIActionsOnTeleportAuth",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
+            "Resource": "arn:aws:dynamodb:eu-west-1:123456789012:table/prod.teleport.auth"
+        },
+        {
+            "Sid": "AllAPIActionsOnTeleportStreams",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
+            "Resource": "arn:aws:dynamodb:eu-west-1:123456789012:table/prod.teleport.auth/stream/*"
+        }
+    ]
+}
```

Comment:
`dynamodb:*` is guaranteed to work, but best practice would be to make this grant the specific permissions needed as we do in our AWS Helm docs (https://goteleport.com/docs/deploy-a-cluster/helm-deployments/aws/):

```js
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "ClusterStateStorage",
            "Effect": "Allow",
            "Action": [
                "dynamodb:BatchWriteItem",
                "dynamodb:UpdateTimeToLive",
                "dynamodb:PutItem",
                "dynamodb:DeleteItem",
                "dynamodb:Scan",
                "dynamodb:Query",
                "dynamodb:DescribeStream",
                "dynamodb:UpdateItem",
                "dynamodb:DescribeTimeToLive",
                "dynamodb:CreateTable",
                "dynamodb:DescribeTable",
                "dynamodb:GetShardIterator",
                "dynamodb:GetItem",
                "dynamodb:UpdateTable",
                "dynamodb:GetRecords",
                "dynamodb:UpdateContinuousBackups"
            ],
            "Resource": [
                "arn:aws:dynamodb:us-west-2:1234567890:table/teleport-helm-backend",
                "arn:aws:dynamodb:us-west-2:1234567890:table/teleport-helm-backend/stream/*"
            ]
        },
        {
            "Sid": "ClusterEventsStorage",
            "Effect": "Allow",
            "Action": [
                "dynamodb:CreateTable",
                "dynamodb:BatchWriteItem",
                "dynamodb:UpdateTimeToLive",
                "dynamodb:PutItem",
                "dynamodb:DescribeTable",
                "dynamodb:DeleteItem",
                "dynamodb:GetItem",
                "dynamodb:Scan",
                "dynamodb:Query",
                "dynamodb:UpdateItem",
                "dynamodb:DescribeTimeToLive",
                "dynamodb:UpdateTable",
                "dynamodb:UpdateContinuousBackups"
            ],
            "Resource": [
                "arn:aws:dynamodb:us-west-2:1234567890:table/teleport-helm-events",
                "arn:aws:dynamodb:us-west-2:1234567890:table/teleport-helm-events/index/*"
            ]
        }
    ]
}
```

Maybe we could turn this into a partial and include it in both?

Reply:
Works for me! This IAM policy was only part of this PR because I added the AWS credentials partial above it, so I'll change this as another PR to keep our history clean

Reply:
Created a PR here: https://github.com/gravitational/teleport/pull/21775